### PR TITLE
OV-62:  create-avatar-page

### PR DESCRIPTION
**Pull request by JKaypa**
Create avatar layout component
image

### CODE REVIEW

File: frontend/src/bundles/common/components/create-avatar/components/dots.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import { Circle } from '../../components.js';
+
+const Dots = (): JSX.Element => {
+    const SIZE = '3px';
+    const COLOR = 'background.600';
```
Comment (sergiy4) [inline frontend/src/bundles/common/components/create-avatar/components/dots.tsx:5]: I think you should create separate files for constants.

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
check out project structure and fix your structure please!

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
also i dont think that is common components

**Review by sergiy4 — COMMENTED** [review]
You should move these files into the 'pages' folder since this is a new page, not just a component.

File: frontend/src/bundles/common/components/create-avatar/components/circles.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Circle } from '../../components.js';
```
Comment (sofiia-trokhymchuk) [inline frontend/src/bundles/common/components/create-avatar/components/circles.tsx:1]: Use relative imports

File: frontend/src/bundles/common/components/create-avatar/components/dots.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import { Circle } from '../../components.js';
```
Comment (sofiia-trokhymchuk) [inline frontend/src/bundles/common/components/create-avatar/components/dots.tsx:1]: Same here

Comment (JKaypa) [reply frontend/src/bundles/common/components/create-avatar/components/dots.tsx:1]: I was thinking at least two levels weren't that bad, but no problem, I'll update it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
DRY

you duplicate all your code 2 times!

Creating avatar will be a separate resource, we can create bundle for it. Also you should add route for this page

**Comment by JKaypa** [comment]
image
image

**Comment by JKaypa** [comment]
image
image

**Comment by stefano-lacorazza** [comment]
Are the children of sidebar the menu items? Because when I add the children they appear on the right of the sidebar.
If not how do I add new menu items?
image

**Comment by JKaypa** [comment]
We should merge [sidebar](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/49) before this one.

**Review by anton-otroshchenko — DISMISSED** [review]

**Comment by nikita-remeslov** [comment]
Please update the target brach. Your PR include changes from another

**Comment by Sanchousina** [comment]
Why there are 0 files changed right now?...

**Comment by JKaypa** [comment]
> Why are there 0 files changed right now?...

I guess because it was merged with sidebar component, and that one was merged to next already

**Comment by o-nedashkivska** [comment]
> > Why are there 0 files changed right now?...
> 
> I guess because it was merged with sidebar component, and that one was merged to next already

You should close the pr if there are no changes

**Comment by JKaypa** [comment]
> > > Why are there 0 files changed right now?...
> > 
> > 
> > I guess because it was merged with sidebar component, and that one was merged to next already
> 
> You should close the pr if there are no changes

Yes, I guess so

**Pull request merged**